services/inventory/services_test.go

```diff
@@ -107,8 +110,123 @@ func TestServices(t *testing.T) {
 		assert.Nil(t, actualService)
 	})
 
+	t.Run("RDSServiceRemoving", func(t *testing.T) {
+		ss, as, ns, teardown, ctx := setup(t)
+		defer teardown(t)
+
+		actualServices, err := ss.List(ctx, models.ServiceFilters{})
+		require.NoError(t, err)
+		require.Len(t, actualServices, 1) // PMM Server PostgreSQL
+
+		as.r.(*mockAgentsRegistry).On("RequestStateUpdate", ctx, "pmm-server")
+		as.vmdb.(*mockPrometheusService).On("RequestConfigurationUpdate")
+		as.r.(*mockAgentsRegistry).On("CheckConnectionToService", ctx,
+			mock.AnythingOfType(reflect.TypeOf(&reform.TX{}).Name()),
+			mock.AnythingOfType(reflect.TypeOf(&models.Service{}).Name()),
+			mock.AnythingOfType(reflect.TypeOf(&models.Agent{}).Name()),
+		).Return(nil)
+
+		node, err := ns.AddRemoteRDSNode(ctx, &inventorypb.AddRemoteRDSNodeRequest{NodeName: "test1", Region: "test-region", Address: "test"})
+		require.NoError(t, err)
+
+		rdsAgent, err := as.AddRDSExporter(ctx, &inventorypb.AddRDSExporterRequest{
+			PmmAgentId:   "pmm-server",
+			NodeId:       node.NodeId,
+			AwsAccessKey: "AKIAIOSFODNN7EXAMPLE",
```

Comment:
let's use some text instead just to avoid false positive report of secrets leakage.

Reply:
Done